Cargo.toml

```diff
@@ -23,8 +23,8 @@ categories = ["command-line-interface"]
 readme = "README.md"
 license = "MIT"
 exclude = ["assets/*", ".github", "Makefile.toml", "CONTRIBUTING.md", "*.log", "tags"]
-edition = "2021"
-rust-version = "1.81.0"
+edition = "2024"
+rust-version = "1.86.0"
```

Comment:
it looks like 1.87.0 will be out in a couple days

https://releases.rs/docs/1.87.0/

so I should be ready to send this for review by next weekend